packages/types/src/interfaces.ts

```diff
@@ -59,9 +59,19 @@ export interface DeployContract {
   code: Uint8Array;
 }
 
+export class Args extends String {
+  toJson() {
+    try {
+      return JSON.parse(Buffer.from(this, 'base64').toString('utf8'));
+    } catch (e) {
+      return String(this);
+    }
+  }
+}
```

Comment:
Types should just provide types and interfaces, not any implementations. 

I also don't think we need to catch the error here. We could get back a string rather than an object.